src/config/getConfig.ts

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-unused-vars */
+import Mainnet from './config.mainnet.json'
+import Testnet from './config.testnet.json'
+import Local from './config.local.json'
+
+export enum SETTINGS {
+  ETHR_DID_CONTRACT = 'ethrDid',
+  RPC_URL = 'rpcUrl'
+}
+
+export const getSetting = (chainId: number, setting: SETTINGS) => {
```

Comment:
This looks weird. Isn't it more clear to have `getDIDRegistryAddress(chainId)` and `getRPCUrl(chainId)`

Reply:
This was future planning as I expected to have many more parameters in the future. I could add getters for them though.